src/main/java/com/github/demo/model/Book.java

```diff
@@ -53,4 +56,18 @@ public String getCover() {
     public void setCover(String cover) {
         this.cover = cover;
     }
+
+    public long getRating() {
+        return rating;
+    }
+
+    public void setRating(long rating) {
+        if (rating < 0) {
+            this.rating = 0;
+        } else if (rating > 5) {
+            this.rating = 5;
+        } else {
+            this.rating = 0;
```

Comment:
This doesn't look okay, should `this.rating` be `rating`? 

```suggestion
            this.rating = rating;
```